src/Bridge/Symfony/Bundle/Resources/config/graphql.xml

```diff
@@ -53,12 +53,12 @@
 
         <service id="api_platform.graphql.resolver.stage.security" class="ApiPlatform\Core\GraphQl\Resolver\Stage\SecurityStage" public="false">
             <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
-            <argument type="service" id="api_platform.security.resource_access_checker" />
+            <argument type="service" id="api_platform.security.resource_access_checker" on-invalid="ignore" />
```

Comment:
Should be `null`, not `ignore` isn't it? Or calling the constructor will still fail.

Reply:
Why test are passing with this configuration? 🤔

Reply:
Because the security component is installed on Travis.

Reply:
No, they behave the same way. The only difference is when inside a method call.

https://symfony.com/doc/current/service_container/optional_dependencies.html#ignoring-missing-dependencies

Reply:
I suggest for consistency's sake, we should not use `on-invalid="null"` except in the specific cases where we need that behaviour (i.e. a method call with `null` value).